iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -1400,7 +1405,7 @@ private static boolean hasParquetNestedTypeWithinListOrMap(Properties tableProps
    * Generates {@link JobContext}s for the OutputCommitter for the specific table.
    * @param configuration The configuration used for as a base of the JobConf
    * @param tableName The name of the table we are planning to commit
-   * @param overwrite If we have to overwrite the existing table or just add the new data
+   * @param branchName If we have to overwrite the existing table or just add the new data
```

Comment:
Need to change the description of `branchName`.

Reply:
done

Reply:
Now i see. Thx.